compiling-KLU.md

```diff
@@ -0,0 +1,157 @@
+# PyBaMM developer install - The KLU sparse solver
+If you wish so simulate large systems such as the 2+1D models, we recommend employing a
+sparse solver.
```

Comment:
I think casadi solver is "sparse" so this could be confusing

Reply:
What is the main motivation for using the KLU solver then? Is it significantly faster?
Could the above sentence be replaced by
```
If you wish so simulate large systems such as the 2+1D models, we recommend employing the KLU sparse solver provided by the Sundials library.
```

Reply:
It's not faster either, just sometimes works when casadi doesn't ... so I guess
```
If you wish to try a different DAE solver, you can install the KLU sparse solver provided by the Sundials library.
```